sonic-slave-buster/Dockerfile.j2

```diff
@@ -395,7 +395,7 @@ RUN export VERSION=1.14.2 \
  && echo 'export PATH=$PATH:$GOROOT/bin' >> /etc/bash.bashrc \
  && rm go$VERSION.linux-*.tar.gz
 
-RUN pip3 install --upgrade pip
+RUN pip3 install --upgrade 'pip==21.0.1'
```

Comment:
With 21.0.1, sonic_yang_models build is broken.
```
----------------------------- Captured stdout call -----------------------------
Failed: pyang -f tree ./yang-models/*.yang > ./yang-models/sonic_yang_tree
----------------------------- Captured stderr call -----------------------------
./yang-models/sonic-acl.yang:8: error: module "ietf-inet-types" not found in search path
```
Can we use version 20.3.4 for buster also?

Reply:
Why do you ask? Can you please provide some reasoning?

Reply:
Build failure is seen for sonic_yang_models package with pip versions higher than 20.3.4.

The error can be seen in https://github.com/Azure/sonic-buildimage/pull/7407 also.

Reply:
Have you verified that pip is the problem here? Why was this error not seen when using pip 21.0, which has been available since January?

Reply:
Yes. I have verified that pip version is the issue here. The build goes through fine when using 20.3.4. With version 21, they made a change in pip that breaks pyang.

Reply:
Let me try again with the latest to ensure the build really goes through

Reply:
My bad! When verifying build for version 21.0.1, I still ended up with 21.1.

The breakage came with 21.1. We are good with 21.0.1.

Sorry.

Reply:
No worries! Good to know. Thanks for confirming!